backend/clubs/models.py

```diff
@@ -1428,6 +1441,36 @@ class Profile(models.Model):
     school = models.ManyToManyField(School, blank=True)
     major = models.ManyToManyField(Major, blank=True)
 
+    def detect_information(self):
+        """
+        Try to detect appropriate values for profile fields based on what platform has
+        returned. Currently only supports detecting the affiliation.
+        This method is not very accurate and should only be used to provide the user
+        an initial guess.
+
+        Overwrites existing information.
+        """
+
+        # detect the affilation
+        if self.user.groups.filter(name="platform_student"):
```

Comment:
Oof, does this check work because an empty queryset is Falsey? Why is python like this

Reply:
Actually I wouldn't bet on this, nice catch! I'm pretty sure this actually doesn't work haha.